cmd/ls.go

```diff
@@ -22,7 +22,7 @@ import (
 	"text/tabwriter"
 
 	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
-	"github.com/dustin/go-humanize"
+	humanize "github.com/dustin/go-humanize"
```

Comment:
Oh yes, as I said you don't need to rename it, it compiles, it is just harder to understand for the reader :). 

Since it is controversial, I will just drop this patch from the series and update the PR. Thanks!